creator-node/src/diskManager.ts

```diff
@@ -167,11 +167,40 @@ export async function computeFilePathInDir(dirName: string, fileName: string) {
       `CID invalid, dirName=${dirName}, fileName=${fileName}, error=${e.toString()}`
     )
     throw new Error(
-      `Please pass in a valid cid to computeFilePathInDir for dirName and fileName. Passed in dirName: ${dirName} fileName: ${fileName} ${e.message}`
+      `Please pass in a valid cid for dirName and fileName. Passed in dirName: ${dirName} fileName: ${fileName} ${e.message}`
     )
   }
+}
+
+/**
+ * Given a directory name and a file name, construct the full file system path for a directory and a folder inside a directory
+ *
+ * eg if you're manually computing the file path to an file `Qmabcxyz` inside a dir `Qmdir123`, use this function to get the
+ * path with both the dir and the file /file_storage/files/r12/Qmdir123/Qmabcxyz
+ * Use `computeFilePath` if you just want to get to the path of a file or directory.
+ *
+ * @param {String} dirName directory name
+ * @param {String} fileName file name
+ */
+export async function computeFilePathInDir(dirName: string, fileName: string) {
+  _validateFileAndDir(dirName, fileName)
+
+  const parentDirPath = computeFilePath(dirName)
+  const absolutePath = path.join(parentDirPath, fileName)
+  genericLogger.info(`File path computed, absolutePath=${absolutePath}`)
```

Comment:
should we have these for a bit on prod as info? or should we change to debug?

Reply:
don't mind some visibility for now during transition